packages/components/pie-modal/src/index.ts

```diff
@@ -330,6 +334,10 @@ export class PieModal extends RtlMixin(PieElement) implements ModalProps {
         }
     }
 
+    private _handleFooterSlotChange () {
```

Comment:
Same here, i think lit should handle the swapping of fallback and default automatically.